include/revng/Pypeline/Helpers/Native/Registry.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Pypeline/Common.h"
+#include "revng/Pypeline/Concepts.h"
+#include "revng/Pypeline/Model.h"
+
+namespace revng::pypeline::helpers::native {
+
+class Container {
```

Comment:
Move `Container` to `ContainerImpl.h` and rename it to `Container.h`.
Same for `Pipe` and `Analysis`.

Reply:
Done